ember_debug/utils/rsvp.js

```diff
@@ -1,21 +1,14 @@
-import Ember from './ember';
+import Ember, { RSVP as emberRSVP } from './ember';
 
-let module, RSVP;
+let module;
 
-try {
-  module = requireModule('rsvp');
-  RSVP = module.default;
-
-  // The RSVP module should have named exports for `Promise`, etc,
-  // but some old versions do not and provide `RSVP.Promise`, etc.
-  if (!('Promise' in module)) {
-    module = RSVP;
-  }
```

Comment:
I don't know what version this code was supporting, nor how the assignment in the if block fixes what the comment says, but with this PR: the RSVP module we import is `emberSafeRequire('rsvp')?.default;` so the content of this block should probably be cleaned up.